tests/unit/test_app.py

```diff
@@ -1,12 +1,13 @@
 from importlib import reload
-from unittest import TestCase
+from unittest import TestCase, mock
 
 from aiohttp.test_utils import AioHTTPTestCase, unittest_run_loop
 from aiohttp.web_app import Application
 from aioresponses import aioresponses
 from envparse import ConfigurationError, env
 
 from app.app import create_app
+from app.security import DEFAULT_RESPONSE_HEADERS
```

Comment:
feels like we should be defining these specifically in the test and not importing from the code